### PR TITLE
[M476b] Append Alan Coral Atlas covariates to field reports

### PR DESCRIPTION
**Pull request by gridcell**

### CODE REVIEW

**Comment by gridcell** [comment]
[separate covars](https://trello.com/c/z7iWHJRx/476-separate-covars)

File: src/api/reports/summary_report.py [hunk 1]
```diff
@@ -37,16 +38,86 @@
...
+        row = row[0:-1 * num_additional_fields]
+        row.extend(covariates)
+        data.append(row)
+    
+    return data
+
+
 def get_viewset_csv_content(view_cls, project_pk, request):
-    request = MockRequest(query_params={"field_report": "t"})
+    request = MockRequest()
```
Comment (saanobhaai) [inline src/api/reports/summary_report.py:100]: Sorry can you remind me why we don't want this param again?

Comment (gridcell) [reply src/api/reports/summary_report.py:100]: Site id is needed to look up the covariates.

**Review by saanobhaai — APPROVED** [review]
Brilliant

**Pull request merged**